internal/gatewayapi/route.go

```diff
@@ -320,12 +320,21 @@ func (t *Translator) processHTTPRouteRule(httpRoute *HTTPRouteContext, ruleIdx i
 			return nil, fmt.Errorf("idle timeout is not supported in envoy gateway")
 		}
 
+		var sessionName string
+		if rule.SessionPersistence.SessionName == nil {
+			// SessionName is optional on the gateway-api, but envoy requires it
+			// so we generate the one here.
+			sessionName = "sticky-host"
```

Comment:
will this value collide ? can we make it unique and use something like this https://github.com/envoyproxy/gateway/blob/866a6cea1af29e0bfbe761fc25b9baddc409ef80/internal/gatewayapi/helpers.go#L395 ?

Reply:
Done.